README.md

```diff
@@ -137,6 +250,24 @@ To start the linter and automatically rectify fixable issues, run:
 make megalint
 ```
 
+#### Django Migrations
+
+Wagtail is built on [Django](https://djangoproject.com/) and changes to its models may require generating and
+running schema migrations. For full details see the [Django documentation on migrations](https://docs.djangoproject.com/en/5.1/topics/migrations/)
+
+Below are the commands you will most commonly use:
+
+```bash
+# Check if you need to generate any new migrations after changes to the model
+django-admin makemigrations --check
+
+# Generate migrations
+django-admin makemigrations
+
+# Apply migrations. Needed if new migrations have been generated (either by you, or via upstream code)
+django-admin migrate
```

Comment:
Let's use poetry, as this will then work both locally (once set up) and in the docker shell.
```suggestion
poetry run django-admin makemigrations --check

# Generate migrations
poetry run django-admin makemigrations

# Apply migrations. Needed if new migrations have been generated (either by you, or via upstream code)
poetry run django-admin migrate
```